src/intake.php

```diff
@@ -13,12 +13,12 @@
 
 require_once "uuid.php";
 
+require_once "check-userid.php";
+
 require_once "check-time.php";
 
 require_once "db-connect.php";
```

Comment:
check-userid.php requires db-connect.php, so moving it up here makes little sense.